configs/tester/tester.toml

```diff
@@ -12,3 +15,53 @@ password = "wJalrXUt"
 
 [context.image]
 name = "cr.backend.ai/multiarch/python:3.13-ubuntu24.04"
+architecture = "x86_64"
+
+[context.session]
+resources = { cpu = 1, mem = "512mb" }
+
+[context.batch-session]
+startup-command = "ls -la"
+
+[context.sse]
+timeout = 60.0
+
+[context.session-template]
+content = """
+[
+  {
+    "id": "c1b8441a-ba46-4a83-8727-de6645f521b4",
+    "is_active": true,
+    "domain_name": "default",
+    "group_id": "2de2b969-1d04-48a6-af16-0bc8adb3c831",
+    "user_uuid": "f38dea23-50fa-42a0-b5ae-338f5f4693f4",
+    "type": "TASK",
+    "name": "python_x86",
+    "template": {
+      "api_version": "6",
+      "kind": "task_template",
+      "metadata": {
+        "name": "cr.backend.ai/multiarch/python",
+        "tag": "3.10-ubuntu20.04"
+      },
+      "spec": {
+        "session_type": "interactive",
+        "kernel": {
+          "image": "cr.backend.ai/multiarch/python:3.10-ubuntu20.04",
+          "environ": {},
+          "architecture": "x86_64",
+          "run": null,
+          "git": null
+        },
+        "scaling_group": "default",
+        "mounts": {
+        },
+        "resources": {
+          "cpu": "2",
+          "mem": "4g"
+        }
+      }
+    }
+  }
+]
+"""
```

Comment:
Please separate the context object so that it doesn't become too large and get passed around.